e2e/test/scenarios/dashboard/tabs.cy.spec.js

```diff
@@ -155,25 +156,51 @@ describe("scenarios > dashboard > tabs", () => {
 
       editDashboard();
       createNewTab();
-      cy.findByRole("tab", { name: "Tab 1" }).click();
+      goToTab("Tab 1");
 
       cy.log("moving dashcards to second tab");
 
       cards.forEach(() => {
-        getDashboardCard(0).realHover();
-        cy.icon("move_card").eq(0).click();
-
-        menu().findByText("Tab 2").click();
+        moveDashCardToTab({ tabName: "Tab 2" });
       });
 
       getDashboardCards().should("have.length", 0);
 
-      cy.findByRole("tab", { name: "Tab 2" }).click();
+      goToTab("Tab 2");
 
       getDashboardCards().should("have.length", cards.length);
     },
   );
 
+  it("should allow moving dashcard even if we don't have permission on that underlying query", () => {
+    const questionDetails = {
+      native: {
+        query: "select 42",
+      },
+      collection_id: ADMIN_PERSONAL_COLLECTION_ID,
+    };
+    cy.createNativeQuestionAndDashboard({
+      questionDetails,
+      dashboardDetails: {
+        collection_id: NORMAL_PERSONAL_COLLECTION_ID,
+      },
+    }).then(({ body: { dashboard_id } }) => {
+      cy.signInAsNormalUser();
+      visitDashboard(dashboard_id);
+    });
+
+    editDashboard();
+    createNewTab();
+
+    goToTab("Tab 1");
+
+    moveDashCardToTab({ tabName: "Tab 2" });
+
+    saveDashboard();
+
+    getDashboardCards().should("have.length", 0);
```

Comment:
maybe we could add a reload after the save to make sure everything is actually saved?
but it would slow down the test a bit